qtconsole/frontend_widget.py

```diff
@@ -728,7 +728,7 @@ def restart_kernel(self, message, now=False):
 
     def append_stream(self, text):
         """Appends text to the output stream."""
-        self._append_plain_text(text, before_prompt=True)
+        self._append_plain_text(text, before_prompt = True)
```

Comment:
I think this could be reverted since the equal is for a `kwarg`. So something like:
```suggestion
        self._append_plain_text(text, before_prompt=True)
```